package.json

```diff
@@ -1,25 +1,26 @@
 {
-    "version": "5.1.0",
+    "version": "5.2.0-alpha.11",
     "private": true,
-    "workspaces": [
-        "packages/*"
-    ],
     "repository": "https://github.com/dhis2/cli",
     "author": "Austin McGee <austin@dhis2.org>",
     "contributors": [
         "Viktor Varland <viktor@dhis2.org>"
     ],
+    "workspaces": [
```

Comment:
this is still needed because `d2 utils release` relies on it to discover the packages of a monorepo, but we should update it at some point to read from pnpm-workspace instead. Hopefully by then, we move to a different of managing releases like [changesets](https://github.com/changesets/changesets) or [rush](https://rushjs.io/) that's more inline with monorepos..